### DISCUSSION TITLE
Add "Flip Horizontally" and "Flip Vertically"

### DISCUSSION THREAD

**Discussion by JohnPlanetary** [mifi/lossless-cut#2186] [Ideas]
These wonderful LossLess Cut program already have "Rotation", but would be very nice to also add "Flip Horizontally" and "Flip Vertically", that I think everyone would agree are just a very ultra basic functions that too many time are needed because of the use of front cameras of smartphones or because people use the rear camera of the smartphone but do it in front of some mirror and that will reverse for example the phrasing in the clothing and other objects that people are using and that sometimes is the main focus.

Something like these exists already for example in the "Paint.NET" that is made for images, but in video:
![flip_example](https://github.com/user-attachments/assets/aa85f173-8b20-4b11-a10d-401ab402be30)
image source: "[Paint.NET](https://github.com/paintdotnet/release)" program option.

I hope these suggestion is not very outside of the scope of the program and is considered easy and essential enough to be possible to add.

**Comment by mifi** [comment]
this has been requested a few times (see issues), but afaik it's not possible to losslessly flip videos

**Comment by JohnPlanetary** [comment]
But it can be done with some lose of quality? Maybe have that option behind some sort of warning that if used the video will loose quality. It probably would still be useful, I mean, people that need that for true professional works where no quality can be lost probably already have their advanced video editing software.

**Reply by mifi** [reply]
yes, it can be done with loss of quality - see faq: https://github.com/mifi/lossless-cut/blob/master/issues.md#faq
